packages/components/src/clipboard-button/index.js

```diff
@@ -31,9 +31,9 @@ export default function ClipboardButton( {
 			return;
 		}
 
-		if ( hasCopied ) {
+		if ( hasCopied && typeof onCopy === 'function' ) {
 			onCopy();
-		} else {
+		} else if ( typeof onFinishCopy === 'function' ) {
```

Comment:
Hm, can we not just check `if ( onFinishCopy )`?

Reply:
We could... but this is the safest way as we call it below and expect a function. It would fail with an wrong truthy provided value

Reply:
Sure, but we're not expecting anything other than a function or undefined. Elsewhere we rarely check if values are of the right type. For example, we're not going to check if a number is of the wrong type. It's actually good if there is an error, then the person passing a wrong value knows to adjust it.

Reply:
Okay. pushed the change :)